api/v1alpha3/azurecluster_conversion.go

```diff
@@ -115,7 +119,7 @@ func (dst *AzureCluster) ConvertFrom(srcRaw conversion.Hub) error { // nolint
 		}
 		dst.Annotations[azureEnvironmentAnnotation] = src.Spec.AzureEnvironment
 	}
-	// Preserve Hub data on down-conversion.
```

Comment:
why remove the comment?

Reply:
Removed it by mistake, Thanks for catching it!